composer.json

```diff
@@ -6,17 +6,17 @@
     "require": {
         "php": "^7.3 || ^8.0",
         "sulu/sulu": "^2.4 || ^2.5@dev",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0"
+        "symfony/config": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/dependency-injection": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/framework-bundle": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/http-foundation": "^4.4 || ^5.4 || ^6.3 || ^7.0",
+        "symfony/http-kernel": "^4.4 || ^5.4 || ^6.3 || ^7.0"
     },
     "require-dev": {
         "coduo/php-matcher": "^5.0 || ^6.0",
         "handcraftedinthealps/code-coverage-checker": "^0.2.1",
         "handcraftedinthealps/zendsearch": "^2.0",
-        "jackalope/jackalope-doctrine-dbal": "^1.3.4",
+        "jackalope/jackalope-doctrine-dbal": "^1.3.4 || ^2.0",
```

Comment:
@TheCadien PS: the jackalope versions need also be upgrade to support Symfony 7 else you still will get Symfony 6.4 for example for symfony/console.